### PR TITLE
feat: Add i18next and i18next-http-backend for new service util

### PR DESCRIPTION
**Pull request by mthomps4**
... Might as well have this setup for when we need it. 

## Changes
- Adds i18n deps 
- Adds Utility helpers 
- Adds Test Server Call 
- Adds test call to Home page 

## Checklist

- [x] Requires dependency update?
- [x] Generating a new app works

### CODE REVIEW

**Comment by mthomps4** [comment]
Update: 
Waiting on Playwright PR to fix E2E tests here...

File: packages/create-bison-app/template/pages/index.tsx
Comment (dennis-campos) [inline packages/create-bison-app/template/pages/index.tsx:31]: Is it throwing an error when removing `any` from `context`? I thought it was inferred from `GetServerSideProps`

**Comment by dennis-campos** [comment]
~~Looks good, Matt! Are we waiting for the test to pass before I can approve or pending another PR review?~~

Nevermind, noticed your previous comment.

**Review by dennis-campos — APPROVED** [review]

**Pull request merged**
